packages/editor/src/components/inserter/index.js

```diff
@@ -96,24 +96,24 @@ export default compose( [
 			getInserterItems,
 			getBlockOrder,
 		} = select( 'core/editor' );
+
 		const insertionPoint = getBlockInsertionPoint();
-		const parentId = rootClientId || insertionPoint.rootClientId;
+		if ( rootClientId === undefined ) {
+			rootClientId = insertionPoint.rootClientId;
+		}
+
 		return {
 			title: getEditedPostAttribute( 'title' ),
-			insertionPoint: {
-				rootClientId: parentId,
-				layout: rootClientId ? layout : insertionPoint.layout,
-				index: rootClientId ? getBlockOrder( rootClientId ).length : insertionPoint.index,
-			},
+			layout: rootClientId ? layout : insertionPoint.layout,
+			index: rootClientId ? getBlockOrder( rootClientId ).length : insertionPoint.index,
```

Comment:
The two lines above are different from what we had previously. As we were not assigning the `insertionPoint.rootClientId` before testing. Not sure if it's impactful.

Reply:
Oh, you're totally right. I expect this is wrong as implemented. I'll plan to fix.